.github/workflows/test.yml

```diff
@@ -444,7 +444,7 @@ jobs:
 
       - name: Run tests
         run: |
-          turbo run check-types test --filter=...[${{ github.event.pull_request.base.sha || 'HEAD^1' }}] --filter="./packages/*" --filter="!@vercel/*" --filter="!turborepo-tests*" --color
+          turbo run check-types test --filter=...[${{ github.event.pull_request.base.sha || 'HEAD^1' }}] --filter="./packages/*" --color
```

Comment:
Was it the use of multiple exclusions that caused the overfit or maybe the exclusion of a pattern? [We document](https://turbo.build/repo/docs/core-concepts/monorepos/filtering#excluding-workspaces) this usage so I would like a better understanding of what we did wrong here. (Also noticed we have 0 test coverage for exclusions 😢 )